### DISCUSSION TITLE
account ui theme

### DISCUSSION THREAD

**Discussion by Ekouyoja** [keycloak/keycloak#15774] [Q&A]
Hello everyone,

Is there a way to try the latest account-ui theme like we can do with admin-ui theme ?
I didn't find a response in the different readme.
Thanks

**Comment by xgp** [comment]
Has anyone found any information on when this will supersede the `keycloak.v2` account UI, and if the new, new implementation in the `keycloak-ui` repo is something fundamentally new, or just a port to use vite and the new build outside of the `keycloak` repo?

**Comment by ssilvert** [comment]
@xgp See answer here: https://github.com/keycloak/keycloak/discussions/11549#discussioncomment-4727984
